source/Engine.cpp

```diff
@@ -2142,7 +2142,7 @@ void Engine::DoCollection(Flotsam &flotsam)
 	if(flotsam.OutfitType())
 	{
 		const Outfit *outfit = flotsam.OutfitType();
-		if(outfit->Get("installable") < 0.)
+		if(outfit->Get("minable") > 0.)
```

Comment:
I'm fine with this breaking change, as it allows content creators to opt-in to displaying an item as mined, while also allowing other items to be mined and also not installable, but not described or made visible as such. (For example, a "broken" outfit from ship debris that might not always be present in a system.)

Reply:
Yeah, in this case, it would mostly be a cosmetic text issue for old plugins, compared to it affecting the behavior of bulk selling in the trading panel. Chances are there are more pressing issues with older plugins in those cases, and it'll be nice flexibility going forward.